dist/adaptive-html.cjs.js

```diff
@@ -333,6 +357,67 @@ rules.image = {
     });
   }
 };
+rules.tableSection = {
+  filter: ['thead', 'tbody', 'tfoot'],
+  replacement: function replacement(content, node) {
+    var fallbackText = 'To view this table content, please open this card in the Guru app';
+    var maxColumns = 3;
+    var maxCellCharacters = 100;
+    var rows = content.length;
+    var columns = (content[0] || {
+      items: []
+    }).items.length;
+
+    if (columns > maxColumns) {
+      return createTextBlock(fallbackText);
+    }
+
+    for (var i = 0; i < rows; i++) {
+      var items = content[i].items || [];
+
+      if (items.some(function (item) {
+        return (item.text || '').length > maxCellCharacters;
```

Comment:
is this saying if the length of  item.text or '' is greater than maxCellCharacters? isnt the length of '' never going to be greater than maxCellCharacters?

Reply:
this is checking the `maxCellCharacters` against text per table cell, which usually looks something like:
```
 {
    "type": "TextBlock",
    "text": "blah blah",
    "wrap": true
 }
```

so by the time the json gets to `rules.tableSection`, `content` looks something like the following as a result of how `rules.tableRow` and `rules.tableCell` are implemented:

```
     [
          {
              "type": "Container",
              "items": [
                  {
                      "type": "TextBlock",
                      "text": "blah blah",
                      "wrap": true
                  },
                  {
                      "type": "Image",
                      "url": "https://blah.com",
                      "altText": "image.jpg"
                  }
              ]
          },
          {
              "type": "Container",
              "items": [
                  {
                      "type": "TextBlock",
                      "text": "blah",
                      "wrap": true
                  },
                  {
                      "type": "TextBlock",
                      "text": "blah",
                      "wrap": true
                  }
              ]
          }
      ]
```